src/components/main-card-component/main-card-component.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import {  Card, Button } from 'react-bootstrap'
+
+import {MdHelpOutline} from 'react-icons/md'
+import { AiOutlineArrowLeft } from "react-icons/ai";
+
+export default function MainCard(props, {children}){
+
+    /* Expecting the arguments
+    CardTitle: The title you want for the card
+    HelpLink: The link to the help documentation for this section
```

Comment:
Tiny doc / comment thing - Looks like the HelpLink was replaced by the doc_extension?

Reply:
Ah brilliant, removed in [this commit](https://github.com/l-gorman/rhomis-data-app/pull/11/commits/a5b00831e65893c930199e83c4ccbb16cd1f4e44)!